permissions/plugin-qualys-was.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "qualys-was"
+github: "jenkinsci/qualys-was"
```

Comment:
The repo is `jenkinsci/qualys-was-plugin` (https://github.com/jenkinsci/qualys-was-plugin), the `-plugin` is required here.